src/ResourceManagement/DataFactory/DataFactoryManagement/DataFactoryManagementChangelog.md

```diff
@@ -1,5 +1,13 @@
 ﻿For additional details on features, see the full [Azure Data Factory Release Notes](https://azure.microsoft.com/en-us/documentation/articles/data-factory-release-notes). 
 
+## Version 4.7.0
```

Comment:
This version number needs to be changed in AssemblyInfo.cs, the nuget.proj and .nuspec files. 

In addition, please squash these 5 commits into one. Here's an article I've found handy in the past: [Git Interactive Rebase, Squash, Amend and Other Ways of Rewriting History](https://robots.thoughtbot.com/git-interactive-rebase-squash-amend-rewriting-history)